panel/applets/icon-tasklist/Application.vala

```diff
@@ -0,0 +1,283 @@
+public class Application: AttentionStatusListener, IconChangeListener, ScreenProvider{
```

Comment:
I will use the same license headers as in the original .vala files. But I have one question. In the files that I created, I'm supposed to put my name and email in the Copyright line, right?

Sorry, I've never done this before.